leshan-integration-tests/src/test/java/org/eclipse/leshan/integration/tests/attributes/WriteAttributeObserveTest.java

```diff
@@ -368,6 +451,214 @@ public void test_with_negative_gap_on_unsigned_integer_resource(Protocol givenPr
+
+        // Setting attribute
+        WriteAttributesResponse writeAttributeResponse = server.send(currentRegistration,
+                new WriteAttributesRequest(TEST_OBJECT, 0, INTEGER_VALUE,
+                        new LwM2mAttributeSet(LwM2mAttributes.create(LwM2mAttributes.MINIMUM_PERIOD, pmin),
+                                LwM2mAttributes.create(LwM2mAttributes.MAXIMUM_PERIOD, pmax))));
+
+        // Verify write attribute response
+        assertThat(writeAttributeResponse).isSuccess();
+
+        // Setting observe
+        ObserveResponse observeResponse = server.send(currentRegistration,
+                new ObserveRequest(TEST_OBJECT, 0, INTEGER_VALUE));
+        assertThat(observeResponse).isSuccess();
+
+        SingleObservation observation = observeResponse.getObservation();
+        assertThat(observation.getPath()).isEqualTo(new LwM2mPath(TEST_OBJECT, 0, INTEGER_VALUE));
+        server.waitForNewObservation(observation);
+
+        // Trigger new notification
+        LwM2mResponse writeResponse = server.send(currentRegistration,
+                new WriteRequest(TEST_OBJECT, 0, INTEGER_VALUE, 30));
+        assertThat(writeResponse).isSuccess();
+
+        // Verify
+        server.ensureNoNotification(observation, (int) (TimeUnit.MILLISECONDS.convert(pmin, TimeUnit.SECONDS) * 0.8),
+                TimeUnit.MILLISECONDS);
+        ObserveResponse response = server.waitForNotificationOf(observation,
+                (int) (TimeUnit.MILLISECONDS.convert(pmin, TimeUnit.SECONDS) * 0.3), TimeUnit.MILLISECONDS);
+        assertThat(response.getContent()).isEqualTo(LwM2mSingleResource.newIntegerResource(INTEGER_VALUE, 30));
+        assertThat(response).hasValidUnderlyingResponseFor(givenServerEndpointProvider);
+    }
+
+    @TestAllTransportLayer
+    public void test_objectId_inheritance(Protocol givenProtocol, String givenClientEndpointProvider,
+                                          String givenServerEndpointProvider) throws InterruptedException {
+
+        Double lt = 500d;
+
+        // Set attribute on objectId
+        WriteAttributesResponse writeAttributeResponse = server.send(currentRegistration,
+                new WriteAttributesRequest(TEST_OBJECT, new LwM2mAttributeSet( //
+                        LwM2mAttributes.create(LwM2mAttributes.MINIMUM_PERIOD, 150l) //
+                )));
+        assertThat(writeAttributeResponse).isSuccess();
+
+        // Set observe relation on TEST_OBJECT
+        ObserveResponse observeResponse = server.send(currentRegistration,
+                new ObserveRequest(TEST_OBJECT, 0, INTEGER_VALUE));
+        assertThat(observeResponse).isSuccess();
+        SingleObservation observation = observeResponse.getObservation();
+        assertThat(observation.getPath()).isEqualTo(new LwM2mPath(TEST_OBJECT, 0, INTEGER_VALUE));
+        server.waitForNewObservation(observation);
+
+        // Change value which doesn't cross the less_than limit (initial value 1024)
```

Comment:
The comment lets think that you test with `lt` but in the code you write `pmin` :point_up:

Reply:
I fixed that